osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerCountdownButton.cs

```diff
@@ -112,7 +113,7 @@ public Popover GetPopover()
                 flow.Add(new RoundedButton
                 {
                     RelativeSizeAxes = Axes.X,
-                    Text = $"Start match in {duration.Humanize()}",
+                    Text = MultiplayerMatchStrings.StartMatchWithCountdown(duration.Humanize()),
```

Comment:
Just noting that the time unit string will not update when changing language while the popover is open, already tracked in https://github.com/ppy/osu/issues/14083:
![osu!_OrDy3Opisj](https://github.com/ppy/osu/assets/35318437/757c63cd-3962-44df-864e-c70290c57971)